### PR TITLE
Fix pylint errors in test and run it in GitHub actions 

### PR DESCRIPTION
**Pull request by JenySadadia**
test: fix pylint errors
 github: enable pylint to run on test files

### CODE REVIEW

File: test/test_listen_handler.py [hunk 1]
```diff
@@ -6,35 +6,13 @@
...
-    return async_mock
-
-
-@pytest.fixture()
-def mock_listen(mocker):
-    async_mock = AsyncMock()
-    mocker.patch('api.pubsub.PubSub.listen',
-                 side_effect=async_mock)
-    return async_mock
+from fastapi.testclient import TestClient
```
Comment (mgalka) [inline test/test_listen_handler.py:13]: Maybe we can use this PR to sort imports alphabetically

File: test/conftest.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2022 Jeny Sadadia
+# Author: Jeny Sadadia <jeny.sadadia@gmail.com>
+
+# pylint: disable=protected-access
+
+"""pytest fixtures for KernelCI API"""
+
+from unittest.mock import AsyncMock
```
Comment (mgalka) [inline test/conftest.py:10]: Put imports in the alphabetical order

**Review by mgalka — CHANGES_REQUESTED** [review]
Just some cosmetic changes

File: test/test_listen_handler.py (hunk 1 above)
Comment (JenySadadia) [reply test/test_listen_handler.py:13]: If I change import order to this,
```
from api.main import app
from api.models import User
from fastapi.testclient import TestClient
```

pylint will complain about wrong import order.
```
************* Module test.test_listen_handler
test/test_listen_handler.py:16:0: C0411: third party import "from fastapi.testclient import TestClient" should be placed before "from api.main import app" (wrong-import-order)

-------------------------------------------------------------------
Your code has been rated at 9.55/10 (previous run: 10.00/10, -0.45)
```

File: test/conftest.py (hunk 2 above)
Comment (JenySadadia) [reply test/conftest.py:10]: The same goes for this too.

Comment (mgalka) [reply test/conftest.py:10]: So, we should group standard library, local imports and third party ones separately and put an empty line between each group.

Comment (JenySadadia) [reply test/conftest.py:10]: Done.

**Pull request merged**